graphql.schema.json

```diff
@@ -294,6 +294,18 @@
             "isDeprecated": false,
             "deprecationReason": null
           },
+          {
```

Comment:
Again just for my education, it looks like this file is auto-generated from the schema. Is there a reason to keep it in source control? Could it be just generated at build time? 

BTW, could we also retrieve the latest schema from keel as part of the build to avoid the issue with discrepancies? It would be another nudge for us to keep keel OSS up to date.

Reply:
It's complicated :)
We don't use this file at the moment so we can also ignore it, but let's put this aside. 

We do need the sdk file in the source code foe the types. However, the schema isn't necessary and we can generate the sdk file directly by querying the backend, in theory. To do so, we'll need to replace the path of the schema in the type generation config with the endpoint of the backend and somehow deal with authentication. It's kind of a PITA so I decided that it's easier to keep it in the source code...

Reply:
I'm still a bit confused. My thought was that, if the only requirement is for the file to be present in source, it could be generated during deck build time from [the latest schema in the keel OSS repo](https://raw.githubusercontent.com/spinnaker/keel/master/keel-web/src/main/resources/schema/schema.graphql), not at runtime. No authentication required.